### ISSUE TITLE
Is it possible to...

### ISSUE THREAD

**Issue by Censtudios** [michaelscheurer/humhub-android#2]
So is it possible to skip the first step where my members would have to insert the URL of my humhub installation? So basically I can have an app specifically for my site. Users download the app and can immediatly login to my humhub without entering the URL.

**Comment by michaelscheurer** [comment]
In this case you had to write your own app. In future I only can imagine, that it exists a database, where each humhub-installation is stored (e.g. automatically when you install a certain module) and the user can get the right humhub-installation by tyiping the first chars of your company/association etc.

Or do you have any other ideas?

I would appreciate to get some help for this undertaking

**Comment by Censtudios** [comment]
I don't have experience in app development, only some PHP, HTML, CSS, Javascript. So I'm not sure how to go about doing this but I can imagine this:

Have some sort of "config" file where if the "default humhub installation" is empty, the user of the app has to manually insert the humhub URL or select from a database like you suggested. But if the config file does have a default installation setup (done by the owner of the humhub installation) then instead of that first screen where the user has to insert the URL, you will see a simple splash screen with a button they have to press (LAUNCH or LOGIN or something like that) to go to the next screen where they actually have to login.

Basically an if/else statement based on the config whether the owner of the app/site has inserted the humhub URL already.

Again, I have no experience in app dev, but since you do I hope this gives you an idea on how to do this.
